app/models/overtime-credit.js

```diff
@@ -1,8 +1,8 @@
 import Model, { attr, belongsTo } from "@ember-data/model";
 
-export default Model.extend({
-  date: attr("django-date"),
-  duration: attr("django-duration"),
-  comment: attr("string", { defaultValue: "" }),
-  user: belongsTo("user"),
-});
+export default class OvertimeCredit extends Model {
+  @attr("django-date") date;
+  @attr("django-duration") duration;
+  @attr("string", { defaultValue: "" }) comment;
```

Comment:
Do you know what the difference between `@attr comment = ""` and `@attr({defaultValue: ""}) comment`?
I tried to google a bit but came up empty handed.

Reply:
:shrug: The [docu does it as well](https://guides.emberjs.com/release/models/defining-models/#toc_options). While using a decorator I would prefer the decorator parameter.

Reply:
Yes i saw that, i would just assume since `field = value` is already defining a default value on a class, this would be the way to go.